softdelete/models.py

```diff
@@ -164,6 +171,11 @@ def set_deleted(self, d):
     def _do_delete(self, changeset, related, force_policy=None):
         rel = related.get_accessor_name()
 
+        # if the policy for this relation is set to SOFT_DELETE
+        # we should just end processing of this relation
+        if self.softdelete_relation_policy.get(rel) == self.SOFT_DELETE:
```

Comment:
I think we have to introduce new policy called: `DO_NOTHING` and place it here, instead of `SOFT_DELETE`. Since `SOFT_DELETE` means that we want delete object, but actually, we want to do nothing with it

Reply:
Yeah, point taken. I tried to reuse as much but your version makes more sense.